core/config/project_settings.cpp

```diff
@@ -362,24 +362,30 @@ bool ProjectSettings::_get(const StringName &p_name, Variant &r_ret) const {
 Variant ProjectSettings::get_setting_with_override(const StringName &p_name) const {
 	_THREAD_SAFE_METHOD_
 
-	StringName name = p_name;
-	if (feature_overrides.has(name)) {
-		const LocalVector<Pair<StringName, StringName>> &overrides = feature_overrides[name];
-		for (uint32_t i = 0; i < overrides.size(); i++) {
-			if (OS::get_singleton()->has_feature(overrides[i].first)) { // Custom features are checked in OS.has_feature() already. No need to check twice.
-				if (props.has(overrides[i].second)) {
-					name = overrides[i].second;
-					break;
-				}
+	const LocalVector<Pair<StringName, StringName>> *overrides = feature_overrides.getptr(p_name);
+	if (overrides) {
+		for (uint32_t i = 0; i < overrides->size(); i++) {
+			if (!OS::get_singleton()->has_feature((*overrides)[i].first)) {
+				continue;
+			}
+
+			// Custom features are checked in OS.has_feature() already. No need to check twice.
+			const RBMap<StringName, VariantContainer>::Element *override_prop = props.find((*overrides)[i].second);
+			if (!override_prop) {
```

Comment:
Replacing 
```
if (!override_prop) {
				continue;
			}

return override_prop->get().variant;
```
with 
```
if (override_prop) {
				return override_prop->get().variant;
			}
```

is just less loc in my opinion (and as seen in previous PR, the first few rounds, when branch predictor has no value for something, it takes the else branch, which is the retrun here, which **sounds** less likely)

Reply:
As mentioned previously, I favor putting the trivial branch in blocks while the nontrivial branch stays in-line - even if it results in more LOC :)

Reply:
I fail to see how this one would be more trivial though (you can close this, of course :) )

Reply:
You're right; in this case it's not as clear as in others.

Maybe this helps:
In the case of a loop, `continue` is usually more trivial than `return`, since `return` is a goal and `continue` means the current element is not what we are looking for.
It's more consistent to write code that has `continue` on the indented block and 'the other' code in non indented blocks. But I can see why you would disagree in this case :)

Reply:
I would also use `if (override_prop) { return override_prop->get().variant; }`, `return`/`break`/`continue` are pretty much same complexity, so it's not adding any readability here. But won't insist on this change.

Reply:
I don't have a super strong opinion on this one, so I'll accommodate and use the shorter positive variant as suggested :)